docs/sql-reserved-and-non-reserved-key-words.md

```diff
@@ -0,0 +1,566 @@
+---
+layout: global
+title: SQL Reserved/Non-Reserved Key Words
+displayTitle: SQL Reserved/Non-Reserved Key Words
+---
+
+When `spark.sql.parser.ansi.enabled` is set to true (false by default), some keywords are reserved for Spark SQL.
```

Comment:
Could you also add a new column? Just document all the keywords that are either reserved or not-reserved when `spark.sql.parser.ansi.enabled` is set to false?

Reply:
Yeah, sure. We should do. I'll update